lib/actions/push.ts

```diff
@@ -193,18 +197,12 @@ export const push: CommandDefinition<
 				Path to a local YAML or JSON file containing Docker registry passwords used to pull base images`,
 		},
 		{
-			signature: 'live',
-			alias: 'l',
+			signature: 'nolive',
 			boolean: true,
 			description: stripIndent`
-				Note this feature is in beta.
-
-				Start a live session with the containers pushed to a local mode device.
-				The project source folder is watched for filesystem events, and changes
-				to files and folders are automatically synchronized to the running
-				containers. The synchronisation is only in one direction, from this machine to
-				the device, and changes made on the device itself may be overwritten.
-				This feature requires a device running supervisor version v9.7.0 or greater.`,
+				Don't run a live instance on this push. The filesystem will not be monitored, and changes
```

Comment:
"live session" instead of "live instance"?

Reply:
Done